rearrange_cars_luiza.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jul 22 10:37:40 2017
+Rearrange the car assignment
+@author: Luiza
+"""
+
+import logging
+import unittest
+import numpy as np
+
+'''Given: There is a parking lot with N spaces and N-1 cars in it.
+The parking lot is described by an array of numbers, that is a permutation of numbers
+from 0 to N. Cars are with numbers from 1 to N-1, 0 means an empty parking space.
+
+E.G. Initial order = [1, 2, 0, 3]
+       Final order = [3, 1, 2, 0]
+
+Task: to rearrange the cars in a given way
+and print a series of moves from position x to y
+
+For our example, the series of moves should be e.g.:
+    move 1 to 2
+    move 3 to 0
+'''
+
+class RearrangeCars(object):
+
+    '''Class to rearrange the cars and print the moves

```

Comment:
Thanks! Having description makes it easier to understand the stream of thought you had when you came up with the solution :)